applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -26,22 +19,20 @@
 import reactor.util.annotation.NonNull;
 
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.EnumSet;
-import java.util.List;
+import java.util.Arrays;
 import java.util.Locale;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 /**
- * Client that contains all the operations for KeyValues in Azure Configuration Store.
+ * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration Store.
```

Comment:
I'm feeling like we need to have a max line length rule, at least for JavaDoc, in CheckStyle.

Reply:
Hmm.. I have my line rules at 120 characters. 🤔 We can enforce one in: .editorconfig if you want. https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length

Reply:
I'm fine with whatever value is set (80, 100, 120, or whatever is fashionable these days)....I just wanted to make sure there was something telling us to wrap eventually! :-)

Reply:
I think scarves are currently stylish. 🤣 120 works for me. I'll update it.

Reply:
![Fashion](https://a1cf74336522e87f135f-2f21ace9a6cf0052456644b80fa06d4f.ssl.cf2.rackcdn.com/images/characters/p-zoolander-will-ferrell.jpg)

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/3320